build.gradle

```diff
@@ -0,0 +1,2 @@
+ext.dockerImageName = 'catalog_kafkatrigger'
```

Comment:
I would recommend updating the naming convention here to either separate words with underscores, or to not use underscores. By looking at the OpenWhisk repo, it looks like the standard practice is to not use underscores.